app/api/account/route.js

```diff
@@ -0,0 +1,36 @@
+import { NextResponse } from 'next/server';
+import connect from "../../../src/utils/data/db";
+import User from "../../../src/models/User";
+import Tasks from "../../../src/models/Tasks";
+
+//retrieve user info
+export const GET = async () => {
+    await connect();
+    const user = await User.findOne({ username: 'johndoe123' });
+
+    try {
+        return NextResponse.json(user);
+    } catch (error) {
+        console.error(error);
+        return NextResponse.json({ message: 'Internal Server Error' }, {status: 500});
+  }
+}
```

Comment:
Get method should have param(body). Such as use userID or username to get the account information

Reply:
Or edit it in later version